# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/python-wheel/use-system-packaging.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Upstream vendors packaging, use system packaging instead

diff --git a/setup.py b/setup.py
index 78b124e..e5025c9 100644
--- a/setup.py
+++ b/setup.py
@@ -4,7 +4,7 @@
 from distutils.core import setup
 
 packages = \
-['wheel', 'wheel.cli', 'wheel.vendored', 'wheel.vendored.packaging']
+['wheel', 'wheel.cli']
 
 package_data = \
 {'': ['*']}
diff --git a/src/wheel/metadata.py b/src/wheel/metadata.py
index b391c96..ffa1706 100644
--- a/src/wheel/metadata.py
+++ b/src/wheel/metadata.py
@@ -12,7 +12,7 @@ from email.message import Message
 from email.parser import Parser
 from typing import Iterator
 
-from .vendored.packaging.requirements import Requirement
+from packaging.requirements import Requirement
 
 
 def _nonblank(str):
diff --git a/tests/test_bdist_wheel.py b/tests/test_bdist_wheel.py
index d202469..95ea27d 100644
--- a/tests/test_bdist_wheel.py
+++ b/tests/test_bdist_wheel.py
@@ -20,7 +20,7 @@ from wheel.bdist_wheel import (
     remove_readonly,
     remove_readonly_exc,
 )
-from wheel.vendored.packaging import tags
+from packaging import tags
 from wheel.wheelfile import WheelFile
 
 DEFAULT_FILES = {
